web-calculator.css

```diff
@@ -0,0 +1,71 @@
+body{
+    background-image: url(https://i.pinimg.com/originals/1e/ef/c5/1eefc5567dc5c55f67ef6849b9235582.jpg), url();
+    background-size: cover;
+    margin:350;
+    position: relative;
+}
+h1{
+    font-family: tahoma, sans-serif;
+    font-size: medium;
+    color:#00FFFF;
+    text-align: center;
+} 
+#a{
+    position: relative;
+    display:block;
+    left:280px;
+    top:-10px;
+}
+#b{
```

Comment:
These should have more meaningful names. Always name things what they are, resist the urge to fluff off names, they're important.